source/retryable-writes/retryable-writes.rst

```diff
@@ -31,9 +31,8 @@ write operation(s) and allow for retrying the operation if the driver fails to
 obtain a command result (e.g. network error after a replica set failover). This
 specification will outline how an API for retryable write operations will be
 implemented in drivers. The specification will define an option to enable
-retryable writes when constructing a client session and describe how a
-transaction ID will be provided to write commands executed within that
-client session.
+retryable writes for an application and describe how a transaction ID will be
+provided to write commands executed therein.
```

Comment:
Apparently you can only comment on lines that are changed... that sucks.

In the paragraph before this one (line 24)  and throughout this document replace DriverSession (and "driver session" etc..) with ClientSession.

The sessionId is no longer opaque (line 23). Drivers must know the format because they can now be generated by the driver.